### ISSUE TITLE
Implement support for role="doc-subtitle"

### ISSUE THREAD

**Issue by stevefaulkner** [FreedomScientific/VFO-standards-support#429]
The ARIA Dpub extension defines a page or section subtitle [`role="doc-subtitle"`](https://www.w3.org/TR/dpub-aria-1.0/#doc-subtitle). This role can be used to identify 

> An explanatory or alternate title for the work, or a section or component within it.

**code example:**
```
<header>
<h1>Chapter 2 The Battle</h1>
<p role="doc-subtitle">Once more unto the breach</p>
</header>
```
Currently the `role="doc-subtitle"` is exposed in IA2 as `role=heading` with no explicit level property, as defined in ARIA 1.1 the default `aria-level` for a heading without an explicit level is `aria-level=2` It is further identified as via iA2 object attribute as `xml-roles:doc-subtitle`. 
Typically screen readers announce the role as _heading level 2_ which does not convey the intent of the `role="doc-subtitle"`.

I suggest that JAWS could implement this to be announced as a "subheading" which appears to be the preference in this ongoing [twitter poll](https://twitter.com/stevefaulkner/status/1295631633723396097?s=20)

**Comment by Heydon** [comment]
+1 

The whole attraction of it is to dissuade folks from misusing headings as subtitles. 

_"Once more unto the breach"_ is not a subheading; it does not label a subsection of the document. It is a supplemental, clarifying accompaniment to the main heading.

**Comment by sarahebourne** [comment]
"Subheading" already has a meaning - it's a level of heading below the one you're referencing.  To use it for something completely different, too, is going to end up causing unnecessary confusion.  "Subtitle" has been used in publishing for centuries and would therefore cause less confusion, so maybe JAWS should just announce it the way it's named.

**Comment by aardrian** [comment]
Two follow-on questions:

1. In cases where the `doc-subtitle` is nested inside a `<h#>`, as below, how should it be announced?
    * "Ramones, heading level 1, Hey Ho Let's Go sub-title"
    * "Ramones Hey Ho Let's Go heading level 1"
2. Can you remove `doc-subtitle` from heading navigation? If it consistently lives adjacent to a heading, it just adds noise in the heading navigation.

```
<h1>Ramones<br>
    <span role="doc-subtitle">Hey! Ho! Let’s Go</span>
</h1>
```

**Comment by scottaohara** [comment]
yes, would not expect this to be exposed as a regular heading at all.  not sure what the implications of that might be for non-web content though...but since this is sort of weirdly crossing the lines, not sure if that'd be an actual issue?

concern i have with doing anything with one dpub role though is that it sets the expectation that other dpub roles would work / be expected in the general concept of web content.
